kubetest/kind/kind.go

```diff
@@ -31,27 +35,46 @@ import (
 )
 
 const (
-	kindBinaryDir = "$HOME/.kubetest/kind"
-	kindNodeImage = "kindest/node:latest"
+	kindBinaryDir       = "$HOME/.kubetest/kind"
+	kindNodeImageLatest = "kindest/node:latest"
+
+	kindBinaryLatest = "latest"
+	kindBinaryBuild  = "build"
 )
 
 var (
 	kindConfigPath = flag.String("kind-config-path", "",
 		"(kind only) Path to the kind configuration file.")
+	kindBinaryTag = flag.String("kind-binary-tag", kindBinaryLatest,
```

Comment:
as discussed in the meeting today, I think we should probably just use a fixed version at known URLs and hashes.